tests/splines/batched_spline_builder.cpp

```diff
@@ -68,6 +101,12 @@ using IDim = std::conditional_t<
         typename GrevillePoints<BSplines<X>>::interpolation_mesh_type,
         ddc::NonUniformPointSampling<X>>;
```

Comment:
```
#elif defined(BSPLINES_TYPE_NON_UNIFORM)
template <typename X>
using BSplines = ddc::NonUniformBSplines<X, s_degree_x>;
template <typename X, typename I>
using IDim = ddc::NonUniformPointSampling<X>;
```
If the BSplines are non-uniform then the sampling will always be non-uniform

Reply:
Yes, this is not what you expect ?